vignettes/adrg.qmd

```diff
@@ -0,0 +1,791 @@
++-----------------------------+-----------------------------------------------+
+| SDTM Controlled Terminology | CDISC SDTM Controlled Terminology, 2022-12-16 |
++-----------------------------+-----------------------------------------------+
+|                             |                                               |
++-----------------------------+-----------------------------------------------+
+| ADaM                        | ADaM-IG v1.1                                  |
++-----------------------------+-----------------------------------------------+
+|                             | ADaM v2.1                                     |
++-----------------------------+-----------------------------------------------+
+|                             |                                               |
++-----------------------------+-----------------------------------------------+
+| ADaM Controlled Terminology | CDISC ADaM Controlled Terminology, 2022-06-24 |
++-----------------------------+-----------------------------------------------+
+|                             |                                               |
++-----------------------------+-----------------------------------------------+
+| Data Definitions            | Define-XML v2.0                               |
++-----------------------------+-----------------------------------------------+
+|                             |                                               |
++-----------------------------+-----------------------------------------------+
+| Medical Events Dictionary   | MedDRA version 8.0                            |
++-----------------------------+-----------------------------------------------+
+
+## Source Data Used for Analysis Dataset Creation
+
+The ADaM datasets were derived from SDTM version 1.2. For traceability, the SDTM is publicly available at the [PHUSE Github Repository](https://github.com/cdisc-org/sdtm-adam-pilot-project/tree/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/tabulations/sdtm).
+
+Which can be traced back to the original [CDISC SDTM & ADaM Pilot Project](https://github.com/cdisc-org/sdtm-adam-pilot-project).
+
+{{< pagebreak >}}
+
+# Protocol Description
+
+## Protocol Number and Title
+
+-   **Protocol Number:** CDISCPilot1
+-   **Protocol Title:** Safety and Efficacy of the Xanomeline Transdermal Therapeutic System (TTS) in Patients with Mild to Moderate Alzheimer’s Disease
+
+The reference documents can be found [here](https://github.com/cdisc-org/sdtm-adam-pilot-project/blob/master/updated-pilot-submission-package/900172/m5/53-clin-stud-rep/535-rep-effic-safety-stud/5351-stud-rep-contr/cdiscpilot01/cdiscpilot01.pdf).
+
+## Protocol Design in Relation to ADaM Concepts
+
+### Objectives:
+
+The objectives of the study were to evaluate the efficacy and safety of transdermal xanomeline, 50cm and 75cm, and placebo in subjects with mild to moderate Alzheimer’s disease.
```

Comment:
I think the power 2 needs to be added for 50 and 75

Reply:
Fixed.